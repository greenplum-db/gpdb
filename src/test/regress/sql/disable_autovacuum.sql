alter system set autovacuum = off;
select * from pg_reload_conf();
