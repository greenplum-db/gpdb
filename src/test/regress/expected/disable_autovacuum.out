alter system set autovacuum = off;
select * from pg_reload_conf();
 pg_reload_conf 
----------------
 t
(1 row)

